network/src/main/java/org/jboss/as/network/SocketBindingManagerImpl.java

```diff
@@ -490,36 +550,30 @@ public void unregisterBinding(String name) {
         }
     }
 
-    static final class UnnamedRegistryImpl implements UnnamedBindingRegistry {
-        private final Map<InetSocketAddress, ManagedBinding> bindings = new ConcurrentHashMap<InetSocketAddress, ManagedBinding>();
+    private static final class UnnamedRegistryImpl implements UnnamedBindingRegistry {
+        private final Map<WrapperBinding, Void> bindings = new ConcurrentHashMap<>();
```

Comment:
Maybe a Set would be better, but we either can't use a `null` value or we need to change the map type. Per the [ConcurrentHashMap JavaDoc](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ConcurrentHashMap.html)

> Like Hashtable but unlike HashMap, this class does not allow null to be used as a key or value.